Makefile

```diff
@@ -14,29 +14,32 @@ BUNDLE_EXEC=bundle exec
 
 default: all
 all: gem deb
-binaries: build/bin/linux-amd64 build/bin/darwin-amd64
+binaries: build/bin/linux-amd64 build/bin/darwin-amd64 build/bin/freebsd-amd64
 gem: $(GEM)
 deb: $(DEB)
 man: $(MANFILES)
 
 build/man/%.gz: man/%.ronn
 	mkdir -p "$(@D)"
-	set -euo pipefail ; $(BUNDLE_EXEC) ronn -r --pipe "$<" | gzip > "$@" || (rm -f "$<" ; false)
+	bash -c 'set -euo pipefail ; $(BUNDLE_EXEC) ronn -r --pipe "$<" | gzip > "$@" || (rm -f "$<" ; false)'
```

Comment:
Can you try changing this line back but adding `SHELL=/bin/bash` near the top of the Makefile?

Reply:
Sounds good to me